### PR TITLE
Added note for SC creation for primera3par backend

### PR DESCRIPTION
**Pull request by sneharai4**
Signed-off-by: Sneha Rai <sneha_srai@yahoo.co.in>

### CODE REVIEW

File: helm/charts/hpe-csi-driver/README.md [hunk 1]
```diff
@@ -96,6 +98,8 @@ helm uninstall hpe-csi --namespace kube-system
 helm delete hpe-csi --purge
 ```
 
+> Note: CRDs are not deleted as part of Helm uninstall.
+
```
Comment (drajen) [inline helm/charts/hpe-csi-driver/README.md:102]: **Note:** Due to a limitation in Helm, CRDs are not deleted as part of the chart uninstall.

File: helm/charts/hpe-csi-driver/README.md [hunk 2]
```diff
@@ -39,6 +39,8 @@ The following table lists the configurable parameters of the HPE-CSI chart and t
 
 It's recommended to create a [values.yaml](https://github.com/hpe-storage/co-deployments/blob/master/helm/values/csi-driver) file from the corresponding release of the chart and edit it to fit the environment the chart is being deployed to. Download and edit [a sample file](https://github.com/hpe-storage/co-deployments/blob/master/helm/values/csi-driver).
 
+> Note: Set `storageClass.create` parameter to `false` for backendType `primera3par`.
+
```
Comment (drajen) [inline helm/charts/hpe-csi-driver/README.md:43]: Can you move this to directly under the table and put it this way:

**Note:** The provided `StorageClass` is currently not compatible with `backendType: primera3par`. How to setup a `StorageClass` for HPE 3PAR and Primera is documented on [SCOD](https://scod.hpedev.io/container_storage_provider/hpe_3par_primera/).

File: helm/charts/hpe-csi-driver/README.md (hunk 1 above)
Comment (sneharai4) [reply helm/charts/hpe-csi-driver/README.md]: Done

File: helm/charts/hpe-csi-driver/README.md (hunk 2 above)
Comment (sneharai4) [reply helm/charts/hpe-csi-driver/README.md]: Done.

**Pull request merged**